clients/flash/shared/ColorPalette.as

```diff
@@ -11,3 +11,19 @@ private const bbbDarkGrey:uint = 0x353B42;
 private const bbbGrey:uint = 0x8B9AA8;
 
 private const bbbWhite:uint = 0xF3F6F9;
+
+// Color names are guessed using http://chir.ag/projects/name-that-color/
+
+private const regentGrey:uint = 0x8A9AA7;

```

Comment:
Calvin is suggesting we use the generic color and add a shade value based on darkness. For example, we can name grey100, grey 200, grey300 with higher numbers darker shade than lower ones.